bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -61,15 +51,7 @@ export default class ActionsBar extends Component {
             close={() => {exitAudio();}}
 
           />
-
-          <Button
-            onClick={this.handleClick}
-            label={'Cam Off'}
-            color={'primary'}
-            icon={'video-off'}
-            size={'lg'}
-            circle={true}
-          />
+          <JoinVideo />
```

Comment:
Because I was told to keep the positioning of the remaining buttons on the actions bar. If I commented it out the other buttons would shift?